springwolf-bindings/springwolf-googlepubsub-binding/src/main/java/io/github/springwolf/bindings/googlepubsub/annotations/channels/GooglePubSubAsyncChannelBinding.java

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-package io.github.springwolf.bindings.googlepubsub.annotations;
+package io.github.springwolf.bindings.googlepubsub.annotations.channels;
```

Comment:
At this point, we keep all public annotations in the same annotations package (per artifact - bindings.googlepubsub in this case) to ease discoverbility of all annotations)

I am happy to discuss it and will check with @sam0r040 next Friday.
@ctasada Do you have an opinion on this?

Reply:
Yes, but the reason I did it is because of `GooglePubSubAsyncSchemaSetting` and `GooglePubSubAsyncMessageSchema`. It can be bit confusing which Schema is for which binding. 
Or may be suggest better naming for these annotations.

Reply:
We prefer consistency and would like to keep the annotation in the same package across all plugins. Also, changing it now introduces a breaking change.